src/test/java/org/folio/innreach/domain/service/impl/ContributionValidationServiceImplTest.java

```diff
@@ -50,6 +53,7 @@
   private static final UUID DO_NOT_CONTRIBUTE_CODE_ID = CRITERIA.getDoNotContributeId();
   private static final String ELIGIBLE_SOURCE = "MARC";
   private static final String INELIGIBLE_SOURCE = "FOLIO";
+  private static final UUID TEST_UUID = UUID.fromString("6802c458-b19e-476f-9187-e8ab7417ecd4");
```

Comment:
what is TEST_UUID? Rename it to something like LOCATION_ID

Reply:
done